Makefile

```diff
@@ -0,0 +1,71 @@
+IMAGE_ORG ?= openebs
+export IMAGE_ORG
+
+DBUILD_DATE      = $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
+DBUILD_REPO_URL ?= https://github.com/openebs/velero-plugin/tree/mayastor
+DBUILD_SITE_URL ?= https://openebs.io
+
+IMAGE      = $(IMAGE_ORG)/velero-plugin-mayastor
```

Comment:
Is it fine as mayastor, or should this be specific for backup?
I guess what I'm asking is, would we have different images for different things or a single image for all?
Ex: 1 for backup, 1 for snapshot streaming, or 1 for both

Reply:
Ideally it should be one image with multiple handlers. So eventually it would be one image per engine. The segregation will happen on the engine level.

Reply:
Ideally as in, this is the velero recommendation/best-practices for their plugins?

Reply:
That's how all the plugins are written for example the gcp and aws one. And it makes sense right, otherwise for 5 usecases we would need 5 init containers.